Program.cs

```diff
@@ -1,100 +1,116 @@
-                    Console.WriteLine("Please enter another number.");
-                    // The data is stored in "num1".
-                    int num2 = Convert.ToInt32(Console.ReadLine());
-                    // Requesting the user's input.
-                    Console.WriteLine("Please enter an arithmetic operator, either addition or subtraction.");
-                    // The operator is stored in "arithmetic"".
-                    var arithmetic = Console.ReadLine();
-                    Console.WriteLine("\n");
+                        // Requesting the user's input for the first number.
+                        Console.WriteLine("\nPlease enter a number.");
+                        // The data is stored in "num1".
+                        if (!int.TryParse(Console.ReadLine(), out int num1))
+                        {
+                            Console.WriteLine("Invalid input. Read please!");
+                            break;
+                        }
+
+                        // Requesting the user's input for the second number.
+                        Console.WriteLine("Please enter another number.");
+                        // The data is stored in "num2".
+                        if (!int.TryParse(Console.ReadLine(), out int num2))
+                        {
+                            Console.WriteLine("Invalid input. Read please!");
+                            break;
+                        }
 
-                    // Addition.
-                    int add = num1 + num2;
-                    // Subtraction
-                    int sub = num1 - num2;
+                        // Requesting the user's input for the arithmetic operator.
+                        Console.WriteLine("Please enter an arithmetic operator, either + for addition or - for subtraction.");
+                        // The operator is stored in "arithmetic".
+                        string arithmetic = Console.ReadLine();
 
-                    // If the operator is "+" addition, else subtraction.
-                    if (arithmetic == "+")
-                    {
-                        Console.WriteLine($"The result of adding {num1} and {num2} is {add}." + "\n");
-                    }
-                    else if (arithmetic == "-")
-                    {
-                        Console.WriteLine($"The result of subtracting {num1} and {num2} is {sub}." + "\n");
-                    }
-                    else
-                    {
-                        Console.WriteLine(arithmetic + " is not a valid operator! Please enter either plus or minus!");
-                    }
-                    break;
+                        // Based on the operator input, we will perform addition or subtraction.
+                        if (arithmetic == "+")
+                        {
+                            int add = num1 + num2;
+                            Console.WriteLine($"The result of adding {num1} and {num2} is {add}.\n");
```

Comment:
Not super important but you can find a way to do this in one line

Reply:
I've fixed this